docs/debug/hdd_furter_tests.md

```diff
@@ -0,0 +1,333 @@
+
```

Comment:
@macpijan in filename s/furter/futher/. BTW this file name is probably no descriptive enough.